Sources/Commands/SwiftTool.swift

```diff
@@ -566,6 +589,28 @@ public class SwiftTool<Options: ToolOptions> {
     func runLLBuild(manifest: AbsolutePath, llbuildTarget: String) throws {
         assert(localFileSystem.isFile(manifest), "llbuild manifest not present: \(manifest.asString)")
         
+        #if canImport(llbuildSwift)
```

Comment:
Rather than have `canImport` in various places in the code, I think it would be nice if we could have an optional protocol defined in one place, and it would be non-nil (and provide the entry points needed) if we have llbuild, and nil otherwise.

Reply:
Will we still need a protocol when llbuildSwift becomes a hard requirement for SwiftPM?

Reply:
No, but it’s probably not much work to add and rip out? If it is, please disregard.